lib/ast-utils/analyze-properties.js

```diff
@@ -65,7 +65,7 @@ var analyzeProperties = function(node) {
               prop.default = val.value;
             }
             else {
-              ;// TODO(atotic): need to make strings out of UnaryExpression, FunctionExpression
+              prop.default = "can't display";// TODO(atotic): need to make strings out of UnaryExpression, FunctionExpression
```

Comment:
Leave default unset when we can't detect it. The API consumer can decide what to do with an undefined default.